static/js/client.js

```diff
@@ -103,14 +103,39 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     }
   }
 
-  function updateTitle() {
+  //see http://stackoverflow.com/a/9609450
+  var decodeEntities = (function() {
+    // this prevents any overhead from creating the object each time
+    var element = document.createElement('div');
+
+    function decodeHTMLEntities (str) {
+      if(str && typeof str === 'string') {
+        // strip script/html tags
+        str = str.replace(/<script[^>]*>([\S\s]*?)<\/script>/gmi, '');
+        str = str.replace(/<\/?\w(?:[^"'>]|"[^"]*"|'[^']*')*>/gmi, '');
+        element.innerHTML = str;
+        str = element.textContent;
+        element.textContent = '';
+      }
+
+      return str;
+    }
+
+    return decodeHTMLEntities;
+  })();
+
+  function updateTitle(message) {
+    //Replace custom title with alert notification text during alarm state. A bit of a hack, but the alerts are getting very confusing as they're often triggered for no apparent reason on the site visualization.
+
+    function s(value, sep) { return value ? value + ' ' : sep || ''; }
 
+	if (!message && !alarmInProgress) {
     var time = latestSGV ? new Date(latestSGV.x).getTime() : (prevSGV ? new Date(prevSGV.x).getTime() : -1)
       , ago = timeAgo(time, browserSettings);
 
     var bg_title = browserSettings.customTitle || '';
 
-    function s(value, sep) { return value ? value + ' ' : sep || ''; }
+    if (browserSettings.customTitle) $('h1.customTitle').text(browserSettings.customTitle);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw '$'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/768506908/issues/source?bid=2452379&fileBranchId=2457543#l138)